### PR TITLE
Add numpydoc  pre-commit For SWE Pt.2

### PR DESCRIPTION
**Pull request by daralynnrhode**
# Add numpydoc Pre-Commit Hook for files contained within swe instrument folder.

## Overview
Adding numpydoc pre-commit. Will be doing this in parts to make it more manageable.  Making any needed changes flagged by the new pre-commit ONLY in files listed within `imap_processing/swe`.

This PR will have changes made in `imap_processing/swe`.
The folders and files listed above should be the only ones that numpydoc pre-commit checks will be run on.
For the most part, almost all the docstrings contained in the given files had to be updated. Most were small changes, others I had to fill in details based on what I could discern.

The main portions that need checked--and most are marked with TODO Double Check or something similar beside them-- are the Parameters and Returns sections. These are the sections I had to do the most additions in.
## Updated Files

-pyproject.toml
-.pre-commit-config.yaml
-imap_processing/swe/utils/swe_utils.py
-imap_processing/swe/l1b/swe_l1b_science.py
-imap_processing/swe/l1b/swe_l1b.py
-imap_processing/swe/l1a/swe_science.py
-imap_processing/swe/l1a/swe_l1a.py
-imap_processing/swe/l0/decom_swe.py
## Testing
-Added numpydoc pre-commits to .pre-commit-config.yaml and pyproject.toml. For now, .pre-commit-config.yaml has exclusions included in it that will be periodically removed as progressed is made in updating all files.

### CODE REVIEW

File: imap_processing/swe/l1a/swe_science.py [hunk 1]
```diff
@@ -95,17 +97,17 @@ def swe_science(decom_data, data_version):
 
     Parameters
     ----------
-    packet_file : str
-        packet file path
+    decom_data : dict
+        Decompressed data. Todo check.
```
Comment (maxinelasp) [inline imap_processing/swe/l1a/swe_science.py]:
```suggestion
        Decompressed packet data.
```

File: imap_processing/swe/utils/swe_utils.py [hunk 2]
```diff
@@ -4,25 +4,26 @@
...
-        SWE data packet
+        SWE data packet.
+    metadata_arrays : dict
+        Metadata arrays.
+
+    Returns
+    -------
     metadata_arrays : dict
-        metadata arrays
+        The metadata_array containing metadata.
```
Comment (maxinelasp) [inline imap_processing/swe/utils/swe_utils.py]:
```suggestion
        The metadata_array with metadata from input data packet.
```

**Review by maxinelasp — APPROVED** [review]
Nice

**Review by bourque — APPROVED** [review]
Looks great!

**Pull request merged**